fc-pattern, sv, tomb, xkill: add examples (#6121)
tldr-pages/tldr: pages/common/xkill.md

- Display a cursor to kill a window when pressing the left mouse button (press any other mouse button to cancel):

`xkill`

> - Display a cursor to kill a window when pressing any mouse button - left, right or middle:

Reviewer: Again dont do lots of pages with completely different content in one PR

> - Display a cursor to kill a window when pressing any mouse button - left, right or middle:
> `xkill -button any`

Reviewer 2: ```suggestion
- Display a cursor to kill a window when pressing any mouse button - left, right or middle:

`xkill -button any`
```

- Display a cursor to kill a window when pressing the left mouse button (press any other mouse button to cancel):

`xkill`

> - Display a cursor to kill a window when pressing any mouse button - left, right or middle:

Reviewer 3: Perhaps this will read better?
```suggestion
- Display a cursor to select a window to kill by pressing any mouse button:
```
